src/components/index.tsx

```diff
@@ -0,0 +1,4 @@
+import Board from './tictactoe board/Board'
```

Comment:
```suggestion
import Board from './TictactoeBoard/Board'
```

Reply:
do not use Space for file or folder names

Reply:
Fixed.